dev/cnf/oss_dependencies.maven

```diff
@@ -619,7 +619,7 @@ org.eclipse.microprofile.openapi:microprofile-openapi-api:1.1.1
 org.eclipse.microprofile.openapi:microprofile-openapi-api:2.0
 org.eclipse.microprofile.openapi:microprofile-openapi-api:3.0
 org.eclipse.microprofile.openapi:microprofile-openapi-api:3.1
-org.eclipse.microprofile.openapi:microprofile-openapi-api:4.0-RC3
+org.eclipse.microprofile.openapi:microprofile-openapi-api:4.0
```

Comment:
The io.openliberty.org.eclipse.rest.client.4.0, io.openliberty.org.eclipse.fault.toleratance.4.1 and io.openliberty.org.eclipse.openapi.4.0 features should also be updated to remove the TODO comment and update the maven coordinates appropriately.

Reply:
Thanks for the heads up